File: java/jse-workspace/ejercicios-con-varias-clases/src/com/pablomonteserin/excepciones/Servicios.java
package com.pablomonteserin.excepciones;

public class Servicios {

	public static int getPrecioConIva(int precio) throws PrecioDemasiadoAltoException{
		if(precio>100)throw new PrecioDemasiadoAltoException();
		return precio;
	}
}

File: c-sharp/ex/exceptions/ejercicio/Servicios.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EjercicioExcepcion
{
  class Servicios
  {
    public double getPrecioConIva(double precio)
    {
      double precioDevuelto = Math.Round(precio * 1.16f,2);
      if(precioDevuelto > 100)
      {
        throw new PrecioDemasiadoAltoException();  
      }
      return precioDevuelto;
    }
  }
}
